src/App.js

```diff
@@ -1,13 +1,23 @@
 import React from "react";
-
+import Heading from "./components/Heading";
 import Bookings from "./Bookings";
+import TouristInfoCards from "./components/TouristInfoCards";
+import Footer from "./components/Footer";
 import "./App.css";
 
 const App = () => {
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      <Heading />
+      <TouristInfoCards />
       <Bookings />
+      <Footer
+        list={[
```

Comment:
Is this the best name for this prop? Does it describe what is actually being passed?

Reply:
yes, I updated it